doc/plugins/org.polarsys.capella.git.doc/html/GitAdapter/GitAdapter.mediawiki

```diff
@@ -1,254 +1,492 @@
+This will open a new view in the Capella environment<br><br>
+[[Image:images/GitStagingView.png|758x300px]]<br><br>
+In this view, we can:
+<ul>
+<li>select files to be committed (by moving them from Unstaged Changes to Staged Changes)</li>
+<li>provide a Commit Message</li>
+<li>provide information about the Author and Committer</li>
+</ul>
+Here we are going to commit all of the files and provide a description "First commit"<br>
+[[Image:images/GitStagingCommit.png|800x300px]]<br><br>
+
+{| border="1"
+|-
+| 
+[[Image:images/Flag.jpg|32x34px]]
+
+
+| 
+When committing a Capella model, all of the files (*.project, *.afm, *.aird and *.melodymodeller) need to be committed consistently in order to avoid corruption of the model!
+|}<br>
+Finally, here we have two different options:
+<ul>
+<li>either Commit (create a new version in the local repository only)</li>
+<li>or Commit and Push (create a new version in the local repository, then push the new version in the remote repository as well)</li>
+</ul>
+We are going to use the second option here.<br>
+When the push is completed, Capella display a confirmation window.<br><br>
+[[Image:images/PushResult.png|600x400px]]<br><br>
+No action is required here and we can select Close.<br>
+{| border="1"
+|-
+|
+[[Image:images/Information.png|48x48px]]
+|
+<ul>
+<li>If we choose "Commit" instead of "Commit and Push" then the new version is only defined in the local repository and the remote repository is not updated.</li>
+<li>To update the remote repository, we can go in the "Git Repositories" view, select origin under Remotes, and use right click, Push.</li>
+<li>This will trigger the push action toward the remote repositor <br><br>
+[[Image:images/RemoteRepo.png|800x300px]]<br><br>
+</li>
+</ul>
+|}
+Now we can see that the icons in the Project Explorer are modified and display a yellow cylinder to show that the files are committed in the repository<br><br>
+[[Image:images/YellowCilinder.png|550x250px]]<br><br>
+Also, the Capella model can now be found in the Git Repositories view in the Working Tree<br><br>
+[[Image:images/WorkingTree.png|540x300px]]<br><br>
+==== Retrieve model from a Git repository ====
+If we initially clone a non-empty repository, we will directly see all its content in the Working Tree.<br>
+If the Working Tree contains Capella models, they can be imported in the Workspace in order to be able to open / modify thembr>
+To do so, you need to select the project in the Working Tree, then use right click, Import Projects<br>
+[[Image:images/ImportProject.png|825x300px]]<br><br>
+
+==== Perform a new commit, retrieve history, compare versions ====
+Now lets modify our Capella model to commit a new version of the model.<br>
+
+Once the model is modify, we will see a ">" icon next to the name of the files in the Project Explorer.<br><br>
+
+This indicate that the files have been modified compared to the last committed version.<br><br>
+[[Image:images/YellowCilinder.png|550x250px]]<br><br>

```

Comment:
Some images don't have a good ratio, for example YellowCilinder.png|550x250px, height is too big.
Check other images too.